stream.go

```diff
@@ -195,7 +195,7 @@ START:
 
 	// Send up to our send window
 	max = min(window, uint32(len(b)))
-	body = bytes.NewReader(b[:max])
+	body = b[:max]
```

Comment:
Before, `body` was a pointer to an object to a new object allocated on the heap: a `bytes.Reader` struct. Now, `body` lives on the stack and is passed on as a value.